lib/statsd.rb

```diff
@@ -328,17 +319,16 @@ def format_event(title, text, opts={})
 
   private
 
-  def escape_event_content(msg)
```

Comment:
Sure thing. I wouldn't prefer this myself, but I'll respect your coding style.

The reason I don't think it should be `escape_event_content` is that the method does not escape anything in the passed String (it used to, but not anymore), but simply returns a new string with the event content escaped.

I'll change the method name back.